File: Converter/Converter/data phisics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Converter
{
    public class data_phisics
    {        

public int CM, M, DM, MM, KM;
public int GRAMM, KG, TONN, CENTN;
public int PA, kPA, MPA, MMrtS;
public int YER, HOUR, MIN, SEC;


public double num;
public double num1;


public double pressureVvod;
public double massVvod;
public double timeVvod;
public double distanceVvod;
public double strenchVvod;
public double energyVvod;
public double temprVvod;

//-------------------------------------
    public const double CEF_MPA = 1000;
    public const double CEF_PA = 1;
    public const double CEF_KPA = 0.001;
    public const double CEF_MM_RT_ST = 0.008;
//---------------------------
    public const double CEF_MM = 10;
    public const double CEF_CM = 1;
    public const double CEF_M = 0.01;
    public const double CEF_KM = 0.0001;
//--------------------------------
    public const double CEF_SEC = 86400;
    public const double CEF_MIN = 1440;
    public const double CEF_HOUR = 24;
    public const double CEF_DAY = 1;
//--------------------------------
    public const double CEF_GM = 1000;
    public const double CEF_KG = 1;
    public const double CEF_CENT = 0.01;
    public const double CEF_TON = 0.001;
//--------------------------------

//--------------------------------
    public const double CEF_KDZH = 0.001;
    public const double CEF_DZH = 1;
    public const double CEF_KK = 0.0002;
    public const double CEF_FF = 0.73;

//-------------------------------------
    public const double CEF_KN =0.001;
    public  const double CEF_N =1;

//-------------------------------------
    public const double CEF_CEL = 1;
    public const double CEF_FAR = 33.8;
    public const double CEF_KELVN =274.15;

   
    }
}

File: Converter/Converter/data phisics.cpp
#include <iostream>
#include <cstdlib>
#include <clocale>
#include <windows.h>
#include <locale>


using namespace std;

int CM, M, DM, MM, KM;
int GRAMM, KG, TONN, CENTN;
int PA, kPA, MPA, MMrtS;
int YER, HOUR, MIN, SEC;

double num;
double num1;


double pressureVvod;
double massVvod;
double timeVvod;
double distanceVvod;
double strenchVvod;
double energyVvod;
double temprVvod;

//-------------------------------------
    const double CEF_MPA = 1000;
    const double CEF_PA = 1;
    const double CEF_KPA = 0.001;
    const double CEF_MM_RT_ST = 0.00750062;
//---------------------------
    const double CEF_MM = 10;
    const double CEF_CM = 1;
    const double CEF_M = 0.01;
    const double CEF_KM = 0.0001;
//--------------------------------
    const double CEF_SEC = 86400;
    const double CEF_MIN = 1440;
    const double CEF_HOUR = 24;
	const double CEF_DAY = 1;
//--------------------------------
    const double CEF_GM = 1000;
    const double CEF_KG = 1;
    const double CEF_CENT = 0.01;
    const double CEF_TON = 0.001;
//--------------------------------

//--------------------------------
    const double CEF_KDZH = 0.001;
    const double CEF_DZH = 1;
    const double CEF_KK = 0.000239006;
    const double CEF_FF = 0.7375620;

//-------------------------------------
    const double CEF_MN =0.001;
    const double CEF_N =1;

//-------------------------------------
    const double CEF_CEL = 1;
    const double CEF_FAR = 33.8;
    const double CEF_KELVN =274.15;


